pulsar-client/src/test/java/org/apache/pulsar/client/impl/MultiTopicsConsumerImplTest.java

```diff
@@ -62,4 +71,41 @@ public void testGetStats() throws Exception {
         impl.getStats();
     }
 
+    @Test
+    public void multiTopicsInDifferentNameSpace() throws PulsarClientException {
+        List<String> topics = new ArrayList<>();
+        topics.add("persistent://public/default/MultiTopics1");
+        topics.add("persistent://public/test-multi/MultiTopics2");
+        topics.add("persistent://public/test-multi/MultiTopics3");
+        ClientConfigurationData conf = new ClientConfigurationData();
+        conf.setServiceUrl("pulsar://localhost:6650");
+
+        PulsarClientImpl clientImpl = new PulsarClientImpl(conf);
+
+        Consumer consumer = clientImpl.newConsumer().topic("persistent://public/test-multi/MultiTopics2")
+                .subscriptionName("multiTopicSubscription")
+                .subscribe();
```

Comment:
I think you need to create a consumer which subscribe multi topics in multi namespaces right? Or can you clarify how this cover your change in this PR.

Reply:
I'll fix it.